src/components/common/PeopleAndRoles/CurrentDirectors.vue

```diff
@@ -31,7 +31,7 @@
             :key="`director:${index}`"
           >
             <!-- Name + Badge -->
-            <td class="text-truncate">
+            <td class="text-truncate px-0">
```

Comment:
By the way, UX/UI decided to remove this truncation/tooltip mechanism in Create UI. Did they say anything about this, here?

Reply:
Oh, they did not. 
How do they handle the really long person/org names?

Reply:
By wrapping.

Maybe ask them about it. It's just taking out a bit of code so pretty easy if needed.

Reply:
Will leave it in for now, as it was a feature as part of Corrections. 
Will bring up during UX review 👍